slither/evm/convert.py

```diff
@@ -0,0 +1,195 @@
+import logging
+import sys
+from slither.core.declarations import (Contract, Function)
+from slither.core.cfg.node import Node
+from slither.utils.function import get_function_id
+from .evm_cfg_builder import load_evm_cfg_builder
+
+logger = logging.getLogger('ConvertToEVM')
+
+class SourceToEVM:
```

Comment:
It looks like `SourceToEVM` is composed only of staticmethod, consider moving to functions rathern than using an object